schema/resource.go

```diff
@@ -91,6 +95,41 @@ func (r *Resource) Columns() []string {
 	return r.Table.Columns.Names()
 }
 
+//nolint:revive
+func (r *Resource) CalculateUniqueValue(consistentID bool) error {
+	if !consistentID {
+		uuidGen := uuid.New()
+		b, err := uuidGen.MarshalBinary()
+		if err != nil {
+			return err
+		}
+		return r.Set(CqIDColumn.Name, b)
+	}
+	names := r.Table.PrimaryKeys()
+	if len(names) == 0 {
+		names = r.Table.Columns.Names()
+	}
+	slices.Sort(names)
```

Comment:
What's the reason we need to sort the PKs? It doesn't guarantee stability

Reply:
It is an aspect of stability... If we rearrange columns (or support users rearranging) then we should not end up with a different hashed value...